modules/statistics/test/TestPlan.md

```diff
@@ -6,13 +6,13 @@
 
 ### Demographic Statistics
 1. Click on the Demographic Statistics tab. For the general statistics table, compare the data in the table to that in the database to ensure that it is being queried correctly.
-2. Check the general statistics filters. Try filtering by both site and project separately. Check that the filters only apply to the general statistics table.
+2. Check the general statistics filters. Try filtering by both site and project separately. Check that the site filter only applies to the general statistics table and that the project filter applies to both tables.
 3. In the breakdown table, try filtering by each of the different instruments. Does each filter appear to be working? Check that it only applies to the breakdown table.
 4. For the sex breakdown, compare the data in the table to that in the database to ensure that it is being queried correctly.
 5. For the sex breakdown, check to see if the % Male is being calculated properly by looking at the data that's in the table.
```

Comment:
These 2 lines could be broken down as 4a and 4b as there are both for the sex breakdown.

Same for lines 2 and 3 in behavioural section